app/src/main/java/org/jellyfin/androidtv/ui/browsing/StdGridFragment.java

```diff
@@ -496,48 +493,43 @@ public void onClick(View v) {
     private JumplistPopup mJumplistPopup;
     class JumplistPopup {
 
-        final int WIDTH = Utils.convertDpToPixel(TvApp.getApplication(), 900);
-        final int HEIGHT = Utils.convertDpToPixel(TvApp.getApplication(), 55);
-
-        PopupWindow mPopup;
-        Activity mActivity;
-        JumpList mJumplist;
-
-        JumplistPopup(Activity activity) {
-            mActivity = activity;
-            LayoutInflater inflater = (LayoutInflater) activity.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
-            View layout = inflater.inflate(R.layout.empty_popup, null);
-            mPopup = new PopupWindow(layout, WIDTH, HEIGHT);
-            mPopup.setFocusable(true);
-            mPopup.setOutsideTouchable(true);
-            mPopup.setBackgroundDrawable(new ColorDrawable(Color.TRANSPARENT)); // necessary for popup to dismiss
-            mPopup.setAnimationStyle(R.style.PopupSlideInTop);
-
-            mJumplist = new JumpList(activity, new CharSelectedListener() {
-                @Override
-                public void onCharSelected(String ch) {
-                    mGridAdapter.setStartLetter(ch);
-                    loadGrid(mRowDef);
-                    dismiss();
-                }
+        private final int WIDTH = Utils.convertDpToPixel(requireContext(), 900);
+        private final int HEIGHT = Utils.convertDpToPixel(requireContext(), 55);
+
+        private final PopupWindow popupWindow;
+        private final AlphaPicker alphaPicker;
+
+        JumplistPopup() {
+            LayoutInflater inflater = LayoutInflater.from(requireContext());
+            View layout = inflater.inflate(R.layout.popup_empty, mGridDock, false);
```

Comment:
Ideally this should also use view binding (PopupEmptyBinding.inflate...) but not necessarily in the scope of this PR as it just modified the existing code.

Reply:
I would say it is out of scope personally. This file is a mess in general. :scream: